packages/bpk-component-datatable/src/BpkDataTable.tsx

```diff
@@ -15,20 +15,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* @flow strict */
 
-import PropTypes from 'prop-types';
 import type { KeyboardEvent } from 'react';
 import { useMemo, useState } from 'react';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
```

Comment:
`react-table` is not typed. There are some community types, but they're not very accurate and are causing issues so went with `any` type whenever there's something from `react-table`